mtags/src/main/scala-2/scala/meta/internal/pc/Keywords.scala

```diff
@@ -20,14 +20,14 @@ trait Keywords { this: MetalsGlobal =>
 
     lazy val notInComment = checkIfNotInComment(pos, text, latestEnclosing)
 
-    lazy val reverseTokens: Iterator[Token] = {
+    lazy val reverseTokens: Array[Token] = {
```

Comment:
It's no longer reverse, no?
```suggestion
    lazy val tokens: Array[Token] = {
```

Reply:
I've changed the place where we reverse everything back to where it was, so tokens are reversed again